docs/data_access.rst

```diff
@@ -0,0 +1,218 @@
+required to use PUDL within Jupyter Notebooks. You can find all our archived data
+products in `the Catalyst Cooperative Community on Zenodo
+<https://zenodo.org/communities/catalyst-cooperative/>`_ .
+
+* The current (beta) version of the archived data and Docker container can be
+  downloaded from `This Zenodo archive <https://sandbox.zenodo.org/record/764417>`_
+* Detailed instructions on how to access the archived PUDL data using a Docker
+  container can be found in our `PUDL Examples repository
+  <https://github.com/catalyst-cooperative/pudl-examples/>`__.
+* The SQLite databases and Parquet files containing the PUDL data, the complete FERC 1
+  database, and EPA CEMS hourly data are contained in that same archive, if you want
+  to access them directly without using PUDL.
+
+.. note::
+
+   If you're already familiar with Docker, you can also pull
+   `the image we use <https://hub.docker.com/r/catalystcoop/pudl-jupyter>`_ to run
+   Jupyter directly:
+
+   .. code-block:: console
+
+      $ docker pull catalystcoop/pudl-jupyter:latest
+
+.. _access-jupyterhub:
+
+---------------------------------------------------------------------------------------
+JupyterHub
+---------------------------------------------------------------------------------------
+
+We've set up a `JupyterHub <https://jupyter.org/hub>`_ in collaboration with
+`2i2c.org <https://2i2c.org>`_ which provides access to all of the processed PUDL
+data and the software environment required to work with it. You don't have to
+download or install anything to use it, but we do need to create an account for you.
+
+* Request an account by submitting `this form <https://forms.gle/TN3GuE2e2mnWoFC4A>`_.
+* Once we've created an account for you
+  `follow this link <https://bit.ly/pudl-examples-01>`_ to log in and open up the first
+  example notebook on the JupyterHub.
+* You can create your own notebooks and upload, save, and download modest amounts of
+  data on the hub.
+
+We can only offer a small amount of memory (4-6GB) and processing power (1 CPU) per
+user on the JupyterHub for free. If you need to work with lots of data or do
+computationally intensive analysis, you may want to look into using the
+:ref:`access-zenodo` option on your own computer. The JupyterHub uses exactly the
+same data and software environment as the Zenodo Archives. Eventually we also want to
+offer paid access to the JupyterHub with plenty of computing power.
+
+.. _access-development:
+
+---------------------------------------------------------------------------------------
+Development Environment
+---------------------------------------------------------------------------------------
+
+If you want to run the PUDL data processing pipeline yourself from scratch, run the
+software tests, or make changes to the source code, you'll need to set up our
+development environment. This is a bit involved, so it has its
+:doc:`own separate documentation <dev/dev_setup>`.
```

Comment:
Given that there is a link to separate documentation, it's unclear what the purpose of the subsections here is and their link to the Development setup specifically. Is it a summary?